src/ui/common/package.json

```diff
@@ -105,5 +105,9 @@
     "tsdx": "^0.14.1",
     "tslib": "^2.4.0",
     "typescript": "^3.9.10"
+  },
+  "dependencies": {
```

Comment:
Could you please move monaco and zip.js to peerDependencies like the rest of the dependencies that we have in common?

Reply:
After done, I'd do a make clean from ui directory and re-install to make sure everything is good.